### ISSUE TITLE
Pyload Beta on DS411 Issue

### ISSUE THREAD

**Issue by combatratcat** [SynoCommunity/spksrc#2224]
Hi guys

Since DSM6.0 i cant get Pyload to work anymore.
When i insatll Pyload Beta after the installation it says it can't start the package.
after a reboot of the whole NAS the Service starts and i get a log that says its starting, running all the Plugins, deactive Plugins, Downloadtime= True
And then it stops.

I wanted to check the pyload user but it seems everything is now running as admin.

I tried several restarts and tried to install the package again and again...
the downloads folder was created an looks as if everything is ok there.

Could you help getting the APP to run again, i'm going nuts.

I can't get a debug log because i cant reach the GUI.

Thanks in advance.
Greetings Marco
##

**Comment by sn0wblind4r** [comment]
There is a Log Button in the "Package-Center" in the installed Apps list.... :)

**Comment by combatratcat** [comment]
hi
i know but you cant set it to DEBUG level from there.

This helped for me for now:
1. Im Paketzentrum "http://packages.synocommunity.com/" als Paketquelle hinzufügen
2. Anschließend das Paket pyLoad (Version 0.4.9-2) installieren. Wichtig: Auf keinen Fall "Nach Installation ausführen" ankreuzen!
3. Im Router die Internet-Verbindung der DS kappen.
4. pyLoad starten. Im Browser diese Adresse aufrufen: https://IP-Adresse-der-DS:8190
5. In den pyLoad-Einstellungen das Plugin "UpdateManager" deaktivieren
6. Alles andere, Pfade etc. nach Wunsch einstellen, auch beim Plugin "ExtractArchive". Im Plugin "HotFolder" den Ordner angeben, in dem die heruntergeladenen DLCs landen
7. Im Router die Internet-Verbindung für die DS wieder zulassen.
8. pyLoad neustarten - fertig!

Non German:
Basically it says you install Pyload and before you run it you disable Internet Connection for the NAS.
Then you connect to Pyload and disable Update in Updatemanager plugin.

then you save, restart and allow internet connection.

Seems to be that the Update that comes in destroys the Installation. Maybe you could predefine this in the package. That would help alot so your installation always works correctly.

Greetings Marco

**Comment by Safihre** [comment]
Further PyLoad updates (like DSM6 compatibility) via #2259.
